examples/orders_create_and_download_multiple_orders.py

```diff
@@ -82,9 +82,9 @@ async def create_and_download(order_detail, directory, client):
 
     # download
     print(f'Downloading {oid} to {directory}.')
-    filenames = await client.download_order(oid, directory, progress_bar=True)
+    filenames, NUM_DOWNLOADS, NUM_DOWNLOADS_SKIPPED = await client.download_order(oid, directory, progress_bar=True)
```

Comment:
I suggest using lower case as these are local variables and upper case suggests global vars

Reply:
Makes sense, done!